libvast/vast/system/actors.hpp

```diff
@@ -201,16 +201,15 @@ using meta_index_actor = typed_actor_fwd<
   caf::replies_to<atom::merge, std::shared_ptr<std::map<
                                  uuid, partition_synopsis>>>::with<atom::ok>,
   // Merge a single partition synopsis.
-  caf::replies_to<atom::merge, uuid, std::shared_ptr<partition_synopsis>>::with< //
-    atom::ok>,
+  caf::replies_to<atom::merge, uuid,
+                  std::shared_ptr<partition_synopsis>>::with<atom::ok>,
```

Comment:
Why did you make this formatting change? The forced line break was intentional to put the reply on the next line.

Reply:
The forced break exceeded the 80 character limit. I can revert that.